src/RoomInvite.js

```diff
@@ -43,10 +43,7 @@ function inviteMultipleToRoom(roomId, addrs) {
 export function showStartChatInviteDialog() {
     const AddressPickerDialog = sdk.getComponent("dialogs.AddressPickerDialog");
 
-    const validAddressTypes = ['mx-user-id'];
-    if (MatrixClientPeg.get().getIdentityServerUrl()) {
-        validAddressTypes.push('email');
-    }
+    const validAddressTypes = ['mx-user-id', 'email'];
```

Comment:
Feel free to inline this into the modal call if you like.

More importantly, `showRoomInviteDialog` below needs the same change.